docs/cimpl6LanguageReference.md

```diff
@@ -78,70 +78,52 @@ Any periods in the _namespace_ should be replaced by dashes. For example, for a
 
 Additional files for configuring and producing FHIR implementation guides (IGs), and their naming conventions include:
 
-* Configuration files: _ig-igname-config.json_
-* Content profile files: _ig-igname-cp.json_
-* IG examples: _ignameExampleName.json_
-* IG HTML pages: _anyName.html_
+* Configuration files: _ig-**myigname**-config.json_
```

Comment:
Please confirm intent - bolded italics = substitution text?  Compare with lines 68-70 above - specifically the "subtopic" and namespace strings.

Reply:
Good catch. Added bold to the subtopic file names and added an explicit notation about bold being what the user substitutes.